code/index.html

```diff
@@ -1,30 +1,104 @@
+                <section class="content">
+            <!-- FAQ -->
+
+            <button class="accordion">WHO AM I?</button>
+            <div class="panel">
+            <p>Hi! I am turning 16 this april and i am currently in ninth grade at Skarpängsskolan, Täby. 
+                This fall I plan to start studying gymnasium level Economics.</p>
+                <p>I come from a big family and have two older brothers and five bonus siblings!</p>
+                <p>I have a lot of previous experience with children from my job as a gymnastics coach for Sollentungymnasterna.</p>
+            </div>
+    
+            <button class="accordion">WHAT CAN I DO?</button>
+            <div class="panel">
+            <p>I will give 100% of my attention to your kids.</p>
+            <p>Sitting will take place at your house, I will get there myself.</p>
+            </div>
+    
+            <button class="accordion">HOW?</button>
+            <div class="panel">
+            <p>Looking forward to hearing from you. Feel free to fill out the form. Tell me a bit about your family, what you would like help with, and I will get back to you soon.</p>
+            <p>References available on request.<br><br>Best wishes //Signe
+            </p>
+            </div>
+        
+        <script src="app.js"></script>
+<!-- S -->           
+                </section>
+
+                <!-- Sidebar -->
+
+
+                <form class="form" action="https://httpbin.org/anything" method=post>
+                    <p>HOW MANY KIDS WILL I SIT FOR?</p>
+                <label class="container">ONE
+                  <input type="radio" checked="checked" name="radio" value="One">
+                  <span class="checkmark"></span>
+                </label>
+                <label class="container">TWO
+                  <input type="radio" name="radio" value="Two">
+                  <span class="checkmark"></span>
+                </label>
+                <label class="container">THREE (Max)
+                  <input type="radio" name="radio" value="Three">
+                  <span class="checkmark"></span>
+                </label>
+                
+                 <label for="fname">NAME
+                 </label><br>
+                  <input type="text" id="fname" name="fname">
+                  
+                 <label for="email">EMAIL
+                 </label>
+                 <input type="email" id="email" name="email" required>
+                 <label for="email">MESSAGE
+                 </label>
+                 <textarea name="message" placeholder="Tell me about your family..."></textarea>
+                 <input type="submit" value="SEND MESSAGE">
+                </form>
+                </div>
+
+            <!-- Footer -->
+            <footer class="footer">
+              
+                <img class="avatar" src="./assets//ninnasquare.png">
+                    <p><a href="mailto:bergman.signe@gmail.com" class="square_btn">E-MAIL</a></p>
```

Comment:
Let's format code over here as well

Reply:
Gave it a try. I hope this is what you mean?